src/key_variants/double_key.rs

```diff
@@ -109,15 +119,15 @@ impl Proof {
         // R_prime = r * G_NUM
         let R = GENERATOR_EXTENDED * r;
         let R_prime = GENERATOR_NUMS_EXTENDED * r;
-        let pair =
+        let public =
```

Comment:
And variable as well. Not going to repeat myself in the rest of the code, you got the gist. :)

Reply:
Fixed, calling it `keys`